src/ui/public/vis/editors/default/resizer.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
+import { uiModules } from 'ui/modules';
+
+uiModules
+  .get('kibana')
+  .directive('visEditorResizer', function () {
+    return {
+      restrict: 'E',
+      link: function ($scope, $el) {
+        $el.on('mousedown', function (event) {
+          const $left = $('.collapsible-sidebar');
+
+          $el.addClass('active');
+          const startWidth = $left.width();
+          const startX = event.pageX;
+
+          function onMove(event) {
+            const newWidth = startWidth + event.pageX - startX;
+            if (newWidth > 360) {
```

Comment:
I'd consider doing this with a CSS rule on collapsible-sidebar. https://github.com/thomasneirynck/kibana/blob/53d81c86ce9b9c7ff74c9d9fc30661e55c25197f/src/ui/public/collapsible_sidebar/collapsible_sidebar.less#L6-L6

Adding `min-width: 360px`

iso enforcing this in JS

Reply:
applying `min-width` to css will break the collapse button (where width is set to 0)